tests/CommunityToolkit.Mvvm.UnitTests/Test_ArgumentNullException.Input.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using CommunityToolkit.Mvvm.Input;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CommunityToolkit.Mvvm.UnitTests;
+
+public partial class Test_ArgumentNullException
+{
+    [TestMethod]
+    public void Test_RelayCommand()
```

Comment:
The test name is only thing that shows up in devops, do we want to add null to the method name somewhere?

![image](https://user-images.githubusercontent.com/24302614/146449649-c3e54286-66c8-4f67-a9a4-d97eee378f4f.png)

Reply:
Yup, I forgot to add the name of the containing type to these new tests, good catch! 😄
Fixed in 7f0c5a1919dbf5dd29d749c840569e8f0628dc1d.